rllib/utils/connector_client.py

```diff
@@ -0,0 +1,153 @@
+"""REST client to interact with a ConnectorServer.
```

Comment:
Should we move all these into the `env` folder? I feel like since `external_env` is already there, `policy_client/server` and `connector_client/server` (or whatever we will eventually call it) should go there as well.